js_modules/dagit/packages/core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -413,36 +409,38 @@ const LaunchAssetChoosePartitionsDialogBody: React.FC<Props> = ({
 
   return (
     <>
-      <DialogBody data-testid={testId('choose-partitions-dialog')}>
-        <Box flex={{direction: 'column', gap: 8}}>
-          {target.type === 'pureAssetBackfill' && (
```

Comment:
@bengotow I forgot to raise this earlier, I didn't know where to put this anchorAsset label. I wasn't sure if it was still necessary in the new design? We may want to add it back